app/src/main/java/com/materiiapps/gloom/Gloom.kt

```diff
@@ -25,7 +28,8 @@ class Gloom : Application() {
                 repositoryModule(),
                 settingsModule(),
                 managerModule(),
-                viewModelModule()
+                viewModelModule(),
+                module { viewModelOf(::MainViewModel) } // Cant group with the rest
```

Comment:
uhhhh why?

Reply:
Bc i can't use the app module as a library in the ui module or redeclare the viewModelModule here due to conflicts